src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -26,12 +26,26 @@ class CachedClient extends Client
      */
     private $caches;
 
+    /**
+     * @var \Closure that accepts a cache key as argument and returns it after sanitizing it.
+     */
+    private $keySanitizer;
+
     public function __construct(FactoryInterface $factory, Connection $conn, array $caches = array())
     {
         parent::__construct($factory, $conn);
 
         $caches['meta'] = isset($caches['meta']) ? $caches['meta'] : new ArrayCache();
         $this->caches = $caches;
+        $this->keySanitizer = function ($cacheKey) { return str_replace(' ', '_', $cacheKey);};
```

Comment:
i was a bit worried about BC breaks, but even if somebody does deploy without resetting the cache, they will just get cache misses, nothing worse.